compose-mac.yaml

```diff
@@ -107,12 +107,29 @@ services:
       - ServiceInsightsDbConnectionString=Server=${DB_CONNECTION},1433;Database=${DB_NAME};User Id=SA;Password=${PASSWORD};TrustServerCertificate=True
 
   # BI Analytics Service
-  create-data-assets:
-    container_name: create-data-assets
+  create-participant-screening-episode:
+    container_name: create-participant-screening-episode
     restart: always
     build:
       context: ./src/
-      dockerfile: ./BIAnalyticsService/CreateDataAssets/Dockerfile
+      dockerfile: ./BIAnalyticsService/CreateParticipantScreeningEpisode/Dockerfile
+    networks:
+      - app-network
+    ports:
+      - "6008:6008"
+    environment:
+      - ASPNETCORE_ENVIRONMENT=Development
+      - FUNCTIONS_WORKER_RUNTIME=dotnet-isolated
+      - ASPNETCORE_URLS=http://*:6008
+      - GetEpisodeUrl=http://get-episode-mgmt:6060/api/GetEpisode
+      - CreateParticipantScreeningEpisodeUrl=http://create-participant-screening-episode:6010/api/CreateParticipantScreeningEpisode
```

Comment:
Will this not call itself? Shouldn't the URL be create-participant-screening-episode-data?

Reply:
No, this one's calling the other CreateParticipantScreeningEpisode function in the BI&A Data service